packages/browserslist-config/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "@workleap/browserslist-config",
+    "author": "Workleap",
+    "description": "Workleap recommended Browserslist config.",
+    "version": "0.0.0",
+    "license": "Apache-2.0",
+    "keywords": [
+        "workleap",
+        "browserslist"
+    ],
+    "contributors": [
+        "Alexandre Asselin <alexandre.asselin@gsoft.com>"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/Workleap/wl-web-configs.git",
+        "directory": "packages/browserslist-config"
+    },
+    "main": "./dist/index.js",
+    "types": "./dist/index.d.ts",
+    "files": [
+        "dist",
+        "CHANGELOG.md",
+        "README.md"
+    ],
+    "scripts": {
+        "build": "tsc",
```

Comment:
If it was a single package in a repo i wouldn't have done it, but now its the same process for every package in the repo, so its consistent.

"slightly" overkill would be more appropriate since its not adding a lot